rtcdatachannel.go

```diff
@@ -147,7 +116,7 @@ func (d *RTCDataChannel) SendOpenChannelMessage() error {
 
 // Send sends the passed message to the DataChannel peer
 func (d *RTCDataChannel) Send(p datachannel.Payload) error {
-	if err := d.rtcPeerConnection.networkManager.SendDataChannelMessage(p, d.ID); err != nil {
+	if err := d.rtcPeerConnection.networkManager.SendDataChannelMessage(p, *d.ID); err != nil {
```

Comment:
To be consistent can/should we follow the NullBool pattern here? 

There are a lot of places through out this PR that do the pointers to primitives, should we do the NullBool pattern instead? I am happy to help with that also if you want to start a branch (we can do the same for the SDP library)

Reply:
Yeah I did this but haven't gotten into this PR yet. Remember I decided to break the PR into smaller PRs because stuff started getting a little too big. :D

In this specific case those it is a little different. This is not because of the NULLBool thingy, it is because the specs, not the config struct but the actual struct, talks about the sctp getting configured through an out-bound channel establishment. This means that after `CreateDataChannel()` the ID may actually be still nil. It is an actual feature.